app/views/welcome/about.html.erb

```diff
@@ -1,2 +1,13 @@
-<h1>Welcome#about</h1>
-<p>Find me in app/views/welcome/about.html.erb</p>
+<!DOCTYPE html>
```

Comment:
You don't need a full HTML page here since it's just a view and it will render inside the application layout.